setup.py

```diff
@@ -67,7 +67,8 @@ def get_packages(package):
         'pyyaml_env_tag>=0.1',
         'importlib_metadata>=3.10',
         'packaging>=20.5',
-        'mergedeep>=1.3.4'
+        'mergedeep>=1.3.4',
+        'mdx_gh_links>=0.2'
```

Comment:
This part of the change is invalid, because MkDocs the library does not depend on `mdx_gh_links`.

`pip install -r requirements/project.txt` is presumed for building docs.

https://github.com/mkdocs/mkdocs/blob/f8b9e967186392d0013a2ca1961a5113e1baf602/requirements/project.txt#L8

In any case, this is most certainly unrelated; I can generally say that adding hugely unrelated changes to your contribution would be frowned upon in almost any project